src/FffGcodeWriter.cpp

```diff
@@ -453,6 +454,7 @@ void FffGcodeWriter::processStartingCode(const SliceDataStorage& storage, const
         const RetractionConfig& retraction_config = storage.retraction_config_per_extruder[start_extruder_nr];
         gcode.writeRetraction(retraction_config);
     }
+    gcode.writeExtrusionModeRequiredByProfile();
```

Comment:
Why is this a separate function?

You can just call `gcode.writeExtrusionMode(meshgroup->getSettingBoolean("relative_extrusion"))`

Reply:
So much for object orientated programming.

OK Tim, i'll change it to make you happy.

Reply:
Tomorrow is my last day in the Cura team, so you'll be rid of me soon enough ;)